### PR TITLE
PR for #84: some examples in help file not working

### LINKED ISSUE #84: Some examples in the help file not working
On August 9, 2022, a user reported that some examples in the help file show message errors. The problem arises because those examples were not updated to account for the `impute` option implemented in version 2.0.0.

### PR DESCRIPTION
**Pull request by Constantino-Carreto-Romero**
Closes #84

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,99 @@ help xtevent
...
+*Besides, impute the policy variable without verifying staggered adoption.
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) cluster(idcode) impute(nuchange)
+            
+*Omit fixed effects
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) cluster(idcode) impute(nuchange) nofe note
+
+*Adjust by estimating a linear trend with gmm method
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
```
Comment (jorpppp) [inline README.md:99]: "Adjust the pre-trend by estimating a linear trend by GMM"

File: README.md [hunk 2]
```diff
@@ -77,7 +77,99 @@ help xtevent
...
+*reghdfe and two-way clustering
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) impute(nuchange) cluster(idcode year) reghdfe ///
+            proxy(wks_work)
+
+
+```
+#### xteventplot
+```
+*setup
```
Comment (jorpppp) [inline README.md:117]: is a stata missing here?

File: README.md [hunk 3]
```diff
@@ -77,7 +77,99 @@ help xtevent
...
+webuse nlswork
+xtset idcode year
+
+*Add an extra effect if union equals 1
```
Comment (jorpppp) [inline README.md:121]: Why do we need this extra union effect?

File: README.md [hunk 4]
```diff
@@ -77,7 +77,99 @@ help xtevent
...
+            pol(union) w(3) vce(cluster idcode) impute(nuchange) ///
+            proxy(wks_work)
+
+*Dependent variable, proxy variable, and overlay plots
+xteventplot, y
+xteventplot, proxy
+xteventplot, overlay(iv)
+```
+#### xteventtest
+```
```
Comment (jorpppp) [inline README.md:147]: missing stata?

File: xtevent/xteventtest.sthlp [hunk 5]
```diff
@@ -87,6 +87,33 @@ the event are equal. #1 must be greater than 0.
...
+{pstd}Test some coefficients to be equal to 0 jointly {p_end}
+{phang2}{cmd:. {stata xteventtest, coefs(1 2)}}{p_end}
+
+{pstd}Test that all pre-event coefficients are equal to 0 {p_end}
+{phang2}{cmd:. {stata xteventtest, allpre}}{p_end}
+
+{pstd}Test that the sum of all pre-event coefficients is equal to 0 {p_end}
+{phang2}{cmd:. {stata xteventtest, allpre cumul}}{p_end}
+
+{pstd}Test wether the coefficients before the event follow a linear trend {p_end}
```
Comment (jorpppp) [inline xtevent/xteventtest.sthlp:110]: Typo

**Comment by jorpppp** [comment]
Thanks @Constantino-Carreto-Romero. I left some comments, can you check them please?

**Comment by Constantino-Carreto-Romero** [comment]
Thanks @jorpppp I will start to check them.

File: README.md (hunk 3 above)
Comment (Constantino-Carreto-Romero) [reply README.md:121]: @jorpppp with the original `ln_wage` variable the plot didn't show any effect in the post-treatment coefficients, and when specifying the `smpath` option in the next examples, the plotted smoothest path was just a horizontal line as shown here:


but adding the extra value, the smoothest path plot is the following:

File: README.md (hunk 2 above)
Comment (Constantino-Carreto-Romero) [reply README.md:117]: Added.

File: README.md (hunk 3 above)
Comment (jorpppp) [reply README.md:121]: @Constantino-Carreto-Romero got it, thanks.

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp I have implemented the changes from your review, thanks.
In a minute I send you the email to answer the user.

**Pull request merged**